### DISCUSSION TITLE
Error occurred prerendering page

### DISCUSSION THREAD

**Discussion by ThomasTomanec** [vercel/next.js#63294] [Help]
### Link to the code that reproduces this issue

https://github.com/ThomasTomanec/portfolio-website

### To Reproduce

Hello, I have a problem. I've created my application using Next.js and I'm trying to deploy it to Vercel. It's giving me this error.

Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error
ApolloError: Response not successful: Received status code 401
    at new ApolloError (/vercel/path0/node_modules/@apollo/client/errors/errors.cjs:33:28)
    at /vercel/path0/node_modules/@apollo/client/core/core.cjs:1994:78
    at both (/vercel/path0/node_modules/@apollo/client/utilities/utilities.cjs:1269:31)
    at /vercel/path0/node_modules/@apollo/client/utilities/utilities.cjs:1260:72
    at new Promise (<anonymous>)
    at Object.then (/vercel/path0/node_modules/@apollo/client/utilities/utilities.cjs:1260:24)
    at Object.error (/vercel/path0/node_modules/@apollo/client/utilities/utilities.cjs:1271:49)
    at notifySubscription (/vercel/path0/node_modules/zen-observable/lib/Observable.js:140:18)
    at onNotify (/vercel/path0/node_modules/zen-observable/lib/Observable.js:179:3)
    at SubscriptionObserver.error (/vercel/path0/node_modules/zen-observable/lib/Observable.js:240:7)

   Generating static pages (2/4) 

   Generating static pages (3/4) 

 ✓ Generating static pages (4/4) 
> Export encountered errors on following paths:
	/
Error: Command "npm run build" exited with 1

### Current vs. Expected behavior

Error occurred prerendering page

### Provide environment information

```bash
idk
```

### Which area(s) are affected? (Select all that apply)

Not sure

### Which stage(s) are affected? (Select all that apply)

Vercel (Deployed)

**Comment by mynameisankit** [comment]
@ThomasTomanec Environment information can be obtained by running `next info` or `yarn next info` as per your package manager

**Comment by ThomasTomanec** [comment]
I don't have a problem with, for example, run dev, but when I want to upload this to vercel, it says this.

**Comment by brunolca** [comment]
It seems like you are using the Github GraphQL API. A HTTP 401 error makes me think you didn't provide a GITHUB_ACCESS_TOKEN in the build environment in Vercel.

**Comment by ThomasTomanec** [comment]
Yes, its true.I have an ".env.local" file on my computer in my project. This file is not in github, but at the same time it contains a key, a token to access the api of my github account. Therefore, I would need to upload this file to git but encrypted, or advise how to add the vercel environment to the project. When I want to add enviroment to vercel, it wants filename and value. I don't know what value means, what they want me to enter from vercel and where I should find it.

**Comment by brunolca** [comment]
I guess you can either add env variables (the same you find in your .env) as key/value pair (in your case GITHUB_ACCESS_TOKEN as the key and the token as value) or you can directly import a .env file containing your environment variavbles
